server/catgenome/src/main/java/com/epam/catgenome/entity/reference/motif/Motif.java

```diff
@@ -38,4 +38,6 @@
     private int end;
     private StrandSerializable strand;
     private String sequence;
+    private String geneId;
```

Comment:
Let's have a List<String> for these fields and introduce a new constructor without gene info to remove code changes in motif iterators